### DISCUSSION TITLE
Help - Login Screen, Realtime, Columns

### DISCUSSION THREAD

**Discussion by GuilhermeHirsch** [Budibase/budibase#1622] [Help] [answered]
Hey guys!

I started using BudiBase recently. It's an amazing tool, congratulations to everyone involved.

I read all the documentation and started development in Builder.

Some questions:
1. Is it possible to edit the login screens? I'm Brazilian, I'm developing tools for my clients, and I need this part to be in Portuguese (to make it easier).
2. Is it possible for a table to update automatically, without having to refresh the page? I have my postgres database in supabase, and I created a webhook in the automation part to be launched in adalo's webapp. I did this but had to refresh the page for the row created to appear in the table.
3. Is it possible to edit the width of a column in the table?
4. Is it possible to edit the column name in Builder?

![Sem título](https://user-images.githubusercontent.com/33108596/120665809-7bee8380-c462-11eb-9be0-4de4254b8785.jpg)

Obs.: I want to adjust this column width to the content width. And I want to change the name from "DATA_DE_EMISSÃO" to "Data de Emissão".

**Comment by joebudi** [comment, marked as answer]
Hey @GuilhermeHirsch 

These are all valid points and thank you for raising them.

1. It is currently not possible to change the content on the login screen but this may be an option in the future - I will certainly raise it with the team.

2. @aptkingston might be best to answer this.

3. It is not possible to edit the width of the column right now.

4. Using an external data source, I don't believe it is possible. @aptkingston might be able to help with this again.

Thanks for reaching out. Interesting that you are Supabase, Budibase, and Adalo - why the need for Adalo?

Joe

**Reply by GuilhermeHirsch** [reply]
I'm an accountant. My business idea is to develop an internal tool (BudiBase) for my client (companies from various sectors) synchronized with the webapp to contact his client. Development on adalo is very fast for this proposal, I can duplicate apps, the monthly fee is unique, it has several integrations ready (stripe, for example).

**Reply by joebudi** [reply]
Awesome. This sounds like an interesting use case. We've actually talked to a few accountants with similar ideas (different parts of the world).

**Reply by GuilhermeHirsch** [reply]
With adalo it is possible to have external collections with the rest api and isolated actions with webhook. Supabase is the best intermediary in my case.

**Reply by GuilhermeHirsch** [reply]
For a while I used the appsheet as an internal tool. But I didn't want to have to use it. That's why I'm here looking at the possibilities. I saw in BudiBase a very promising tool, and I'm sure it would make a difference in my business.

**Reply by slfhstr** [reply]
> Awesome. This sounds like an interesting use case. We've actually talked to a few accountants with similar ideas (different parts of the world).

I know the focus of Budibase is internal apps for an organisation.
Personally I think this is limited-thinking.  There is huge scope for Budibase for 'extranet' uses, rather than purely 'intranet'.
In some respects Budibase V0.9 has gone backwards from V0.8 on this.
I would urge you to reconsider this positioning, with some minor changes it would open up Budibase to a far bigger market.

**Reply by mjashanks** [reply]
@timconsidine I realise that this might seem counterintuitive, but a "Far bigger market" is exactly what we don't need. 

We need focus, so that we can know what to build and know who exactly our users are. Budibase has taken a massive leap for intranet use, even if that means leaving the extranet behind somewhat.

**Reply by slfhstr** [reply]
OK, your choice
But perhaps you should be clearer on website then.
"If you're considering  building an extranet, Budibase is not the tool for you.  Pick something else.
We're not looking to support applications with an extranet model".
Should keep the focus nice and tight.

**Reply by ryzam** [reply]
HI @joebudi , I really exited waiting when budibase will allow login and portal landing page can be customized. I know budibase focusing mainly for internal apps but there is a need for budibase developer to customize such as a labeling that can support multi language in login/portal screen.
